config/velocity_smoother_params.yaml

```diff
@@ -0,0 +1,21 @@
+# Example configuration:
+# - velocity limits are around a 10% above the physical limits
+# - acceleration limits are just low enough to avoid jerking
+velocity_smoother_node:
+  ros__parameters:
```

Comment:
These two fields (`velocity_smoother_node` and `ros_parameters`) seem to be fairly redundant, is it actually needed?

Reply:
Yeah, they are needed for when you launch a node pointing at the parameter file.  The node name is definitely needed, since we support having the parameter for multiple nodes in the same file.  I don't know why `ros__parameters` is there, but it has always been necessary :man_shrugging: .

Reply:
Ah, it's a special snowflake. How about dropping those two fields and just pulling it in like you do with the composable launcher so we have the same style across launchers.

However, leave a comment about it informing users.

Reply:
Done in ec95262

Reply:
I would have preferred additionally dropping the `velocity_smoother` and `ros_parameters` usage in code and yaml (and leaving a comment back in the yaml) so there is no 'magic/seemingly redundant' usage in the code, but I'm nitpicking now. This will do.